app/src/main/java/com/bronzeswordstudios/nasaimageviewer/model/DataResult.java

```diff
@@ -0,0 +1,59 @@
+package com.bronzeswordstudios.nasaimageviewer.model;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+import java.util.ArrayList;
+
+public class DataResult {
+
+	@SerializedName("collection")
+	@Expose
+	private Collection collection;
+
+
+
+	public DataResult (Collection collection) {
+		this.collection = collection;
+	}
+
+
+
+	public Collection getCollection () {
+		return collection;
+	}
+
+
+
+	public void setCollection (Collection collection) {
+		this.collection = collection;
+	}
+
+
+
+	public ArrayList<NasaImage> getNasaImages () {
+		ArrayList<NasaImage> nasaImages = new ArrayList<NasaImage>();
+		for (Item item : this.collection.getItems()) {
+			try {
+				NasaImage nasaImage = new NasaImage(
```

Comment:
This part right here to me seems a little sloppy. You have all data, Why don't you just use the classes you have? The names of the classes you can readjust. The serialization on data fields is just based on the Retrofit annotations, if that makes sense.

Reply:
Addressed in commit 4589813!